crates/core/Cargo.toml

```diff
@@ -45,3 +45,6 @@ foundry-config = { git = "https://github.com/foundry-rs/foundry", branch = "mast
 dojo-test-utils = { workspace = true }
 starknet-crypto = { workspace = true }
 toml = "0.7.5"
+tracing-subscriber = "0.3.17"
```

Comment:
should we have this in workspace instead?

Reply:
I'm not too sure either, I placed it here because it's not used by any other crate than `core` but can move it if considered better practice.

Reply:
we do have `use tracing_subscriber::util::SubscriberInitExt;` in `crates/eth-rpc/src/main.rs`